### PR TITLE
Network layer updates

### PR DESCRIPTION
**Pull request by utsabc**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by sanpj2292 — DISMISSED** [review]
LGTM

**Review by saikumarrs — COMMENTED** [review]
LGTM

File: v0/destinations/braze/util.js [hunk 1]
```diff
@@ -49,4 +58,12 @@ const responseTransform = (destinationResponse, _dest) => {
   };
 };
 
-module.exports = { responseTransform };
+const BrazeNetworkHandler = function() {
```
[the remark this answers is not in the document]
Comment (utsabc) [reply v0/destinations/braze/util.js]: This is addressed

File: adapters/networkHandlerFactory.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const { BrazeNetworkHandler } = require("../v0/destinations/braze/util");
+const { MarketoNetworkHandler } = require("../v0/destinations/marketo/util");
+const {
+  GenericNetworkHandler
+} = require("./networkhandler/genericNetworkHandler");
```
[the remark this answers is not in the document]
Comment (utsabc) [reply adapters/networkHandlerFactory.js]: This is addressed

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**